packages/rest/test/integration/rest.server.integration.ts

```diff
@@ -114,8 +115,61 @@ describe('RestServer (integration)', () => {
       .expect(200, content);
   });
 
+  it('allows static assets to be mounted on multiple paths', async () => {
+    const root = ASSETS;
+    const server = await givenAServer({
+      rest: {
+        port: 0,
+      },
+    });
+
+    server.static('/html-0', root);
+    server.static('/html-1', root);
+
+    const content = fs
+      .readFileSync(path.join(root, 'index.html'))
+      .toString('utf-8');
+    await createClientForHandler(server.requestHandler)
+      .get('/html-0/index.html')
+      .expect('Content-Type', /text\/html/)
+      .expect(200, content);
+    await createClientForHandler(server.requestHandler)
+      .get('/html-1/index.html')
+      .expect('Content-Type', /text\/html/)
+      .expect(200, content);
+  });
+
+  it('allows static assets to be mounted on the same path multiple times', async () => {
```

Comment:
Let's improve this test name:

`merges different static asset directories when mounted on the same path`